scripts/content.js

```diff
@@ -13,8 +13,7 @@ async function isBlue() {
 
   checkmark.click();
 
-  if (blueText) return true;
-  return false;
+  return !!blueText;
```

Comment:
Since we're already changing how we're doing this, I was thinking this would make sense?
```suggestion
  return typeof blueText === "undefined";
```
Personally I think this seems more explicit and readable, but whatever fits the code you're already writing is amazing. You got this!

Reply:
**Read, review and suggest code with empathy.** Personally I like to say "we" and "us" once I've started reviewing and have anything to say. In that way I'm not enforcing anything, but as the button says, merely making a suggestion. ❤️

Reply:
But ofc. if you're me, and talking to... me, I should still be nice and say YOU GOT THIS BOY! 🚀

Reply:
Thumbs down without comment about why isn't helpful.